index.html

```diff
@@ -39,7 +41,88 @@ <h1 class="salute">I'm Alexandra. Glad to see you!</h1>
+                        <li><img class="counter" src="Counter.svg" alt="counter"></li>
+                        <li><p class="text_li">2015</p></li>
+                    </ul>
+                    <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="categories">
+                        <li><p>HTML</p></li>
+                        <li><p>CSS</p></li>
+                        <li><p>JavaScript</p></li>
+                    </ul>
+                    <a class="see_project_button" href="#">See project</a>
+                </div>
+                
+            </div>
+        </section>
+        <section id="stories1">
+            <div class="card">
+                stories1
+            </div>
+        </section>
+        <section id="tonic2">
+            <div class="card">
+                tonic2
+            </div>
+        </section>
+        <section id="stories2">
+            <div class="card">
+                stories2
+            </div>
+        </section>
+    </div>    
+    <section class="about_my_self">
+        
+        <div class="content content_about_self">
+            <h1 class="salute">About Myself</h1>
+            <p class ="description">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <p class="lets_connect">LET’S CONNECT</p>
+            <ul class="icons">
+                <li id="twitter_icon">
+                    <a  href="#"><img class="social" src="SocialMediaIcons\Twitter_icon.png" alt="Twitter_icon"></a>
+                </li>
+                <li id="linkedin_icon">
+                    <a  href="#"><img class="social" src="SocialMediaIcons\Linkedin_icon.png" alt="Linkedin_icon"></a>
+                </li>
+                <li id="m_icon">
+                    <a href="#"><img class="social" src="SocialMediaIcons\M_icon.png" alt="M"></a>
+                </li>
+                <li id="git_icon">
+                    <a href="#"><img class="social" src="SocialMediaIcons\Github_icon.png" alt="Github_icon"></a>
+                </li>
+                <li id="hand_icon">
+                    <a href="#"><img class="social" src="SocialMediaIcons\Hand_icon.png" alt="Hand_icon"></a>
+                </li>
+            </ul>
+            <a class="get_resume_button" href="#">Get my resume</a>
+            <ul class= "content_about_self2">
+                <li class="list_info"><p>Languaje</p></li> 
+                <li class="list_lang"><p>JavaScript</p></li>
+                <li class="list_lang"><p>HTML</p></li>
+                <li class="list_lang"><p>CSS</p></li>
+                <li class="list_info"><p>frameworks</p></li>
+                <li class="list_info"><p>Skills</p></li>
+            </ul>

```

Comment:
- [ ] Your about myself section is not matched with the Figma design. Kindly work on it. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![yourdesign_about_myself](https://user-images.githubusercontent.com/70483835/227507367-936e88b3-78a9-49a9-9dff-abb7aabc2836.png) | ![figma_about_myself](https://user-images.githubusercontent.com/70483835/227507359-89a0579a-02c6-47a3-8010-9ffa041b84f6.png)